src/notiflix.js

```diff
@@ -1923,6 +1923,10 @@
         if (typeof delay !== 'number') { delay = 0; }
         NotiflixBlockUnblockElement(false, selector, false, false, false, delay); // false => hide/remove
       },
+      // Check if the specified element is being blocked
+      isBlocked: function (selector) {
+        return document.querySelector(`${selectors} [id^="${blockSettings.ID}"]`)? true:false;
+      },
```

Comment:
Hello @yhchiu .

Thanks for the contributions to the Notiflix.

Firstly, I am working to update the [Contribution](https://github.com/notiflix/Notiflix/blob/master/CONTRIBUTING.md) standards for the Notiflix. Contributing is not acceptable right now. Anyway;

1- Notiflix uses ECMAScript 5. So, we can not use the Template Literals, etc. because of the Internet Explorer compatibility. 
```js
...querySelector(selector + 'string')...
```
---

2- Notiflix uses UMD. So, we have to use "`window.document`". The "`document`" is not equal to the "`window.document`" all the times.
```js
window.document.querySelector(....
```

---

3- You are trying to select with "`blockSettings.ID`" but it is not defined yet. So you can not use the options like that.

---

4- We have to check the "`selector`" parameter is a "`string`" or not for the user experience and preventing the errors.

---

5- Notiflix uses Eslint and Stylelint for development. Your codes not following these rules. 
Please use the command below to check the lint. (Node.js required)
```sh
npm run notiflix-src-lint
```

---

6- As I said; the Contributing standards not ready yet so; you have to change only "`/src/notiflix.js`" and "`/src/notiflix.css`" for the changes. All other files will be created automatically using Node.js.
So, you do not have to change the `/src/notiflix-aio.js` file. This file will be created automatically also.


So,

I am closing your PR for these reasons.

Thanks for using the Notiflix.

Furkan.